### ISSUE TITLE
Run issue (No module named 'colorama')

### ISSUE THREAD

**Issue by martinlindhe** [orf/gping#5]
```
$ pip3 install pinggraph
Collecting pinggraph
  Downloading pinggraph-0.0.4-py3-none-any.whl
Installing collected packages: pinggraph
Successfully installed pinggraph-0.0.4

$ gping ping.sunet.se
Traceback (most recent call last):
  File "/usr/local/bin/gping", line 7, in <module>
    from gping.pinger import run
  File "/usr/local/lib/python3.5/site-packages/gping/pinger.py", line 10, in <module>
    from colorama import Fore, init
ImportError: No module named 'colorama'
```

(OSX 10.11, python 3.5.0)

**Comment by EntilZha** [comment]
I just opened a pull request to fix that here: #4 

You can fix it with `pip install colorama` though.

**Comment by orf** [comment]
Damn, I knew there would be something. I'm on a train now so can't do much
until later, but thanks for reporting!
